pkg/scalers/beanstalkd_scaler.go

```diff
@@ -34,12 +34,12 @@ type BeanstalkdScaler struct {
 }
 
 type BeanstalkdMetadata struct {
-	Server          string  `keda:"name=server,          order=triggerMetadata"`
-	Tube            string  `keda:"name=tube,            order=triggerMetadata"`
-	Value           float64 `keda:"name=value,           order=triggerMetadata"`
-	ActivationValue float64 `keda:"name=activationValue, order=triggerMetadata, optional"`
-	IncludeDelayed  bool    `keda:"name=includeDelayed,  order=triggerMetadata, optional"`
-	Timeout         uint    `keda:"name=timeout,         order=triggerMetadata, default=30"`
+	Server          string        `keda:"name=server,          order=triggerMetadata"`
+	Tube            string        `keda:"name=tube,            order=triggerMetadata"`
+	Value           float64       `keda:"name=value,           order=triggerMetadata"`
+	ActivationValue float64       `keda:"name=activationValue, order=triggerMetadata, optional"`
+	IncludeDelayed  bool          `keda:"name=includeDelayed,  order=triggerMetadata, optional"`
+	Timeout         time.Duration `keda:"name=timeout,         order=triggerMetadata, default=30"`
```

Comment:
nit: how do you feel about explicitly defaulting to `time.Duration` format instead?
```suggestion
	Timeout         time.Duration `keda:"name=timeout,         order=triggerMetadata, default=30s"`
```